cmd/gh-classroom/clone/student-repos/student-repos.go

```diff
@@ -53,6 +53,10 @@ func NewCmdStudentRepo(f *cmdutil.Factory) *cobra.Command {
 					log.Fatal(err)
 				}
 			}
+			
+			// Default getAll to true unless page is set differently from default.
+			getAll = !cmd.Flags().Changed("page")
+
 			var acceptedAssignmentList classroom.AcceptedAssignmentList
 			if getAll {
 				acceptedAssignmentList, err = shared.ListAllAcceptedAssignments(client, assignmentId, perPage)
```

Comment:
I know this is outside of the change you're making, but does it matter how many we get `perPage` if we're getting them all? What does it mean to get _all_ Accepted Assignments by page? @jalafel

Reply:
@octosteve Yes I honestly have no idea about the previous code! Perhaps it's worthwhile to think this more thoroughly and remove the perPage entirely? I believe the logic differentiation between all and not all was added in [this PR](https://github.com/github/gh-classroom/pull/37) and probably just needs some iteration!